### ISSUE TITLE
The configured user limit (128) on the number of inotify instances has been reached

### ISSUE THREAD

**Issue by timothyparez** [ingvar1995/Camelot#3]
When the app starts it throws an IOException:

> An exception of type 'System.IO.IOException' occurred in System.IO.FileSystem.Watcher.dll but was not handled in user code: 'The configured user limit (128) on the number of inotify instances has been reached, or the per-process limit on the number of open file descriptors has been reached.'

**Info:**
- Source: `Camelot.FileSystemWatcherWrapper/Implementations/FileSystemWatcherWrapper.cs`
- Line: 29
- Commit: 3acd8a643731d5aebcbaa6858011659e8b11da2c
- OS: Ubuntu 20.04
- .NET Core Version: 3.1.201

**Comment by ingvar1995** [comment]
Hello. Seems that inotify limit is low on your machine and it breaks C# `FileSystemWatcher`. As a temporary workaround you can increase inotify limit as described [here](https://github.com/guard/listen/wiki/Increasing-the-amount-of-inotify-watchers). I will think how to solve this and probably will get rid of `FileSystemWatcher` at all

**Comment by timothyparez** [comment]
Yes, might be better since you want this to be usable by end-users.  
Nice project by the way. Good showcase of Avalonia and a good alternative to TotalCommander and MuCommander.
